src/runtime/server.ts

```diff
@@ -26,21 +26,22 @@ const DEFAULT_PORT = process.env.PORT || 3000;
 const RELOAD_DEBOUNCE_MS = 250;
 const DEFAULT_BODY_SIZE_LAMBDA = '6mb';
 
-function requireUncached(module: string): any {
-  delete require.cache[require.resolve(module)];
-  return require(module);
+function loadTwilioFunction(fnPath: string): ServerlessFunctionSignature {
+  return require(fnPath).handler;
 }
 
-function loadTwilioFunction(
-  fnPath: string,
-  config: StartCliConfig
-): ServerlessFunctionSignature {
-  if (config.live) {
-    debug('Uncached loading of %s', fnPath);
-    return requireUncached(fnPath).handler;
-  } else {
-    return require(fnPath).handler;
-  }
+function requireCacheCleaner(
+  req: ExpressRequest,
+  res: ExpressResponse,
+  next: NextFunction
+) {
+  debug('Deleting require cache');
+  Object.keys(require.cache).forEach(key => {
+    if (!(key.endsWith('.node') || key.includes('twilio-run'))) {
```

Comment:
what are the things that end with `.node`? :)

Reply:
Could you add a comment here just to explain why we would want to keep those?

Reply:
I will add a comment, yes. But things that end in `.node` are native extensions and as I was looking into this discovered plenty of places that said that unloading native extensions from the cache has undefined behaviour and could cause problems.